package.json

```diff
@@ -51,7 +51,8 @@
     "lint:types": "tsc --noEmit --jsx react && tsc --noEmit --project ./tsconfig.module_system.json",
     "lint:style": "stylelint \"res/css/**/*.scss\"",
     "test": "jest",
-    "coverage": "yarn test --coverage"
+    "coverage": "yarn test --coverage",
+    "prune:code": "node ./scripts/prune_code.js"
```

Comment:
While the utility we use behind the scenes is `ts-prune` I find the name „prune“ for this script misleading. It does actually not prune code. What about naming it something like „analyse:unused-exports“.

Reply:
I was struggling with the naming a bit :sweat_smile: Your suggestion sounds great - thank you!